toolkit/scripts/srpm_pack.mk

```diff
@@ -84,7 +83,11 @@ $(STATUS_FLAGS_DIR)/build_srpms.flag: $(local_specs) $(local_spec_dirs) $(local_
 $(STATUS_FLAGS_DIR)/build_toolchain_srpms.flag: $(STATUS_FLAGS_DIR)/build_srpms.flag
 	@touch $@
 else
-$(STATUS_FLAGS_DIR)/build_srpms.flag: $(chroot_worker) $(local_specs) $(local_spec_dirs) $(SPECS_DIR) $(go-srpmpacker) $(srpm_pack_list_file) $(local_spec_sources)
+
+# Dependencies common to both the full packer and the toolchain-only packer targets
+common_srpm_packer_deps = $(local_specs) $(local_spec_dirs) $(SPECS_DIR) $(local_spec_sources) $(go-srpmpacker) $(built_srpms) $(BUILD_SRPMS_DIR)
```

Comment:
Since make tracks timestamps of files, and we pick which files to track via `find`, it will lose track of a file if it is deleted. But... deleting the file will update the timestamp of the parent directory. If we include the directory it will trigger a rebuild if one of the files is removed.